server/jetstream_cluster.go

```diff
@@ -833,7 +833,13 @@ func (js *jetStream) setupMetaGroup() error {
 	atomic.StoreInt32(&js.clustered, 1)
 	c.registerWithAccount(sacc)
 
-	js.srv.startGoRoutine(js.monitorCluster)
+	js.srv.startGoRoutine(
+		js.monitorCluster,
+		map[string]string{
```

Comment:
Hmm yea this seems worse. Maybe back to strings but concat the key and value?

```go
js.srv.startGoRoutine( js.monitorCluster, "type: metaleader", ..)
```

Reply:
Only thing with this approach is that it might end up ugly for other reasons, i.e. `fmt.Sprintf` littered everywhere:
```
js.srv.startGoRoutine( js.monitorCluster, "type:metaleader", fmt.Sprintf("stream:%s", streamName), ...)
```

Reply:
Hmm, true.